src/instance/InstanceController.ts

```diff
@@ -139,4 +140,48 @@ export class InstanceController {
       res.status(errorMessage.code).json(errorMessage);
     }
   }
+
+  @Post(':instanceId/clear')
+  @ApiOperation({summary: 'Clear a conversation.'})
+  @ApiResponse({description: 'The conversation has been cleared.', status: 200})
+  @ApiResponse({description: 'Instance not found', status: 404})
+  @ApiResponse({description: 'Validation error', status: 422})
+  @ApiResponse({description: 'Internal server error', status: 500})
```

Comment:
You may need `{description: 'Internal server error', status: 500}` more often so you can put it in a separate file and import if needed to not write the description again and again.